chunks/modelfreemodels.ipynb

```diff
@@ -337,11 +338,34 @@
     }
    },
    "source": [
+    "## Exercise\n",
```

Comment:
I suggest phrasing this as inferences in the homework problems (or examples from class). Could just be me, but when confronted by "anything from class", I have trouble thinking of one, whereas there are only a few in the HW to remember.

(Also the latest example from missing data is a terrible candidate for this.)